backend/app/schemas.py

```
@@ -106,53 +122,51 @@
-    size: Optional[str]
-    quantity: int
-
-
-class BillCreate(BaseModel):
-    amount: float
-    method: str
-    trx_id: str
-    status: str
+        
+        
+class UserUpdate(BaseModel):
```

Ruff (0.8.2)

Redefinition of unused `UserUpdate` from line 61

(F811)